jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -241,12 +242,11 @@ def _on_add_data_to_viewer(self, msg):
                             else:
                                 # default to pix2 if input data is not in surface brightness units
                                 # TODO: for cubeviz, should we check the cube itself?
-                                self.angle_unit.selected = 'pix2'
+                                self.angle_unit = 'pix2'
```

Comment:
this will overwrite the `Select` with a string (maybe that is the answer to my own question above, but I don't think we want to destroy the `Select`)